### PR TITLE
Fixed invalid uri exception.

### PR DESCRIPTION
**Pull request by diepsohung**
Fix issue #59

### CODE REVIEW

File: lib/3llo/command/board.rb [hunk 1]
```diff
@@ -21,9 +21,7 @@ def execute(subcommand, args)
         else
           handle_invalid_subcommand(subcommand, args)
         end
-      rescue InvalidCommandError => exception
-        Command::Board::Invalid.execute(exception.message)
-      rescue InvalidArgumentError => exception
+      rescue InvalidCommandError, InvalidArgumentError, URI::InvalidURIError => exception
```
Comment (qcam) [inline lib/3llo/command/board.rb]: I think I would handle the invalid ID here instead.

https://github.com/qcam/3llo/blob/master/lib/3llo/command/board/select.rb#L24

File: .rubocop.yml [hunk 2]
```diff
@@ -57,8 +57,14 @@ Security/JSONLoad:
 Style/Alias:
   EnforcedStyle: prefer_alias
 
-Style/BracesAroundHashParameters:
-  Enabled: false
+Style/HashEachMethods:
```
Comment (qcam) [inline .rubocop.yml]: Could you move unrelated change to another PR and elaborate why it's important to add? Thanks.

Comment (diepsohung) [reply .rubocop.yml]: I've opened the PR #61 for it.

File: lib/3llo/command/board.rb (hunk 1 above)
Comment (diepsohung) [reply lib/3llo/command/board.rb]: It would be more effective when we handle [here](https://github.com/qcam/3llo/blob/master/lib/3llo/command.rb#L68) because I realized some commands that take the ID argument will cause crashes.

Comment (qcam) [reply lib/3llo/command/board.rb]: In that case, I think we should handle the exception at where it was raised: https://github.com/qcam/3llo/blob/master/lib/3llo/remote_server.rb#L80.

Comment (diepsohung) [reply lib/3llo/command/board.rb]: Nice, I've fixed using Regexp, please check it out.

File: lib/3llo/remote_server.rb [hunk 3]
```diff
@@ -77,6 +77,10 @@ def delete(req_path, req_headers, payload, expected_codes = EXPECTED_CODES)
     private
 
     def build_request_uri(req_path)
+      req_uri = endpoint_url + req_path
+      invalid_uri = (req_uri =~ /\A#{URI::DEFAULT_PARSER.make_regexp}\z/).nil?
+      raise InvalidCommandError.new("Invalid command.") if invalid_uri
+
       URI.parse(endpoint_url + req_path)
```
Comment (qcam) [inline lib/3llo/remote_server.rb]:
```rb
def build_request_uri(req_path)
  URI.parse(endpoint_url + req_path)
rescue URI::Error
  raise InvalidArgumentError, "invalid command arguments"
end
```

Comment (qcam) [reply lib/3llo/remote_server.rb]: Ideally we should introduce a new exception under `RemoteServer` but I think this is fine for now.

Comment (diepsohung) [reply lib/3llo/remote_server.rb]: Well nice!  Such a short and simple way, thank you! I've fixed it, please review again.
